doc/specifications/rest/REST-API-V2.rst

```diff
@@ -2062,6 +2062,10 @@ or if destination is /content/trash
         HTTP/1.1 201 Created
         Location: /content/trash/<ID>
 
+.. code:: http
```

Comment:
I see "or if destination is /content/trash", and then two different http responses - I'm missing a mention of why there are two possibilities for this case.